### ISSUE TITLE
Man page contains misleading information

### ISSUE THREAD

**Issue by mbukatov** [skyrings/skynet#4]
There are few issues with current manpage:
- `SEE ALSO` section refers to `service(8)` man page discussing System V init scripts - but skyring provides systemd service
- `BUGS` section states that there are "no known bugs" - I'm not sure if such statement is either correct or useful
- `SYNOPSIS` section mentions `[start/stop/reload/restart]` arguments - weren't they dropped?

**Comment by mbukatov** [comment]
ping

**Comment by cybernth** [comment]
Yes you are right.The man pages were not updated after we moved from init scripts to systemd. This needs to be updated
